packages/contracts/package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@eth-optimism/rollup-contracts",
   "private": true,
-  "version": "0.0.1-alpha.27",
+  "version": "0.0.1-alpha.29",
   "main": "build/index.js",
   "files": [
     "build/**/*.js",
+    "build/contracts/*",
```

Comment:
What do we export these for btw?

Reply:
for the mock cross domain contracts -> https://github.com/ethereum-optimism/Buidler-Deposit-Withdraw-Example/blob/abb04077f1f239b78d5677af3f2fdcf1f545ce9e/contracts/ERC20/DepositedERC20.sol#L4